pkg/webui/console/lib/device-utils.js

```diff
@@ -34,6 +34,10 @@ export const PHY_V1_0_2_REV_B = { value: 'PHY_V1_0_2_REV_B', label: 'PHY V1.0.2
 export const PHY_V1_0_3_REV_A = { value: 'PHY_V1_0_3_REV_A', label: 'PHY V1.0.3 REV A' }
 export const PHY_V1_1_REV_A = { value: 'PHY_V1_1_REV_A', label: 'PHY V1.1 REV A' }
 export const PHY_V1_1_REV_B = { value: 'PHY_V1_1_REV_B', label: 'PHY V1.1 REV B' }
```

Comment:
I think we want to rename these now as well. I think we should explicitly call them RP001, with the exception of 1.0 and 1.0.1 which are TS001.

@adriansmares can confirm the labels to use here.

Reply:
What about MAC versions? Should I rename them as well?

Reply:
Indeed, `LoRaWAN Specification 1.x.y` is the naming scheme we could use there.